planner/core/rule_join_elimination.go

```diff
@@ -179,6 +179,8 @@ func (o *outerJoinEliminator) doOptimize(p LogicalPlan, aggCols []*expression.Co
 		aggCols = newCols
 	}
 
+	aggCols = append(aggCols, p.Schema().Columns...)
```

Comment:
This fix seems incorrect, why not using https://github.com/erjiaqing/tidb/commit/70d1676889f3b47fbfab7197cf0020b72e4aecbb? `isParentColsAllFromOuterTable == true` is a pre-requisite for outer join elimination, so I think we should check it firstly.